graylog2-web-interface/src/views/components/widgets/FieldsConfiguration.tsx

```diff
@@ -77,6 +130,9 @@ FieldsConfiguration.defaultProps = {
   selectSize: undefined,
   menuPortalTarget: undefined,
   testPrefix: '',
+  showSelectAllRest: false,
+  showDeSelectAll: false,
+  showListCollapseButton: false,
 };
```

Comment:
It would be nice to improve the spacings a bit.

<img width="269" alt="image" src="https://github.com/Graylog2/graylog2-server/assets/46300478/eb7b4939-6982-44ed-90c5-4a1a03771949">


There is more space between "Hide 3 selected fields" and "timestamp" than between the "Fields to export" and "Messages limit" section. There is little space between the "Add a field" input and the buttons below.